packages/ember-engines/tests/integration/engine-resolver-test.js

```diff
@@ -1,19 +1,19 @@
-import { module, test } from "qunit";
-import { setupRenderingTest } from "ember-qunit";
-import { render } from "@ember/test-helpers";
-import hbs from "htmlbars-inline-precompile";
-import engineResolverFor from "ember-engines/test-support/engine-resolver-for";
+// import { module, test } from "qunit";
```

Comment:
@villander should we simply delete this file or uncomment this?

Reply:
I have to uncomment these lines since we are going to release this API, but we don't want to remove the old ones yet. We going to delete the older API progressively once we migrate our internal tests here. Does it make sense?